templates/django_registration/registration_form.html

```diff
@@ -5,7 +5,69 @@ <h1 class="display-4">Sign up</h1>
 
 <form method="post" action="{% url "django_registration_register" %}">
   {% csrf_token %}
-  {{ form|crispy }}
+
+  <div id="div_{{ form.username.id_for_label }}" class="form-group{% if form.username.field.required %} required{% endif %}">
```

Comment:
Whoof.

Reply:
Yeah. I'm not sure how to specify things like "add this form prefix" in the Python declaration of the form, so I just went with jamming it into the template. Not thrilled with it, but probably worth getting out sooner than not.